packages/babel-preset-env/test/fixtures/preset-options-babel-7/rhino-1_7/input.mjs

```diff
@@ -0,0 +1,2 @@
+const a = () => 1;
```

Comment:
Can you also add something that Rhino supports?

```suggestion
const a = () => 1;

0b01;
```

Also, this file should now be renamed to `1_7_3`

Reply:
ah of course

Reply:
well actually that's a great point i didn't even think about - rhino should support all of that so something is wrong here

Reply:
According to `compat-table`, it doesn't fully support arrow functions because it doesn't pass these tests:
- `lexical "super" binding in constructor`
- `lexical "super" binding in method`
- `lexical "new.target" bindin`

However, it doesn't pass them not because of arrow functions but because it doesn't support classes. Maybe we could exclude those three tests in https://github.com/babel/babel/blob/5145c98a735d8ea49fbaffd3d81033a6882bf570/packages/babel-compat-data/scripts/data/plugin-features.js#L35-L37 by explicitly listing all the other arrow tests? Similar to https://github.com/babel/babel/blob/5145c98a735d8ea49fbaffd3d81033a6882bf570/packages/babel-compat-data/scripts/data/plugin-features.js#L66-L73.

Those three removed tests can then be added to https://github.com/babel/babel/blob/5145c98a735d8ea49fbaffd3d81033a6882bf570/packages/babel-compat-data/scripts/data/plugin-features.js#L41-L43 and https://github.com/babel/babel/blob/5145c98a735d8ea49fbaffd3d81033a6882bf570/packages/babel-compat-data/scripts/data/plugin-features.js#L93-L95

Reply:
Right that's all implemented now!